code/modules/research/designs/wiremod_designs.dm

```diff
@@ -238,6 +238,20 @@
 	id = "comp_module"
 	build_path = /obj/item/circuit_component/module
 
+/datum/design/component/get_hid
+	name = "HID Request Component"
```

Comment:
Rename "HID" and everything else to just "hardware ID", or whatever it's short for. This confused me when you said it originally, and I had to dig through to see what it meant.